### PR TITLE
Add python constant and enable user set values for conv1d init method

### PR DESCRIPTION
**Pull request by dding3**

### CODE REVIEW

File: pyzoo/zoo/pipeline/api/autograd.py [hunk 1]
```diff
@@ -484,6 +484,21 @@ def set_weight(self, value):
...
+    A constant Variable without weights.
+    :param data: value of the Variable.
+    :param name: Optional. Name of the Variable
+    """
+    def __init__(self, data, name=None):
+        self.data = data
+        super(Constant, self).__init__(None, JTensor.from_ndarray(data), name)
+
+    @property
+    def shape(self):
```
Comment (zhichao-li) [inline pyzoo/zoo/pipeline/api/autograd.py]: can we use the `shape` which exists in `VariableOperator` directly?

Comment (dding3) [reply pyzoo/zoo/pipeline/api/autograd.py]: Updated

**Comment by dding3** [comment]
jekins passed. http://172.168.2.101:8080/view/ZOO-PR/job/ZOO-PR-Validation/2020/

**Comment by dding3** [comment]
Thanks for your review.

**Pull request merged**